packages/contracts/deployments/ethereum/.chainId

```diff
@@ -0,0 +1 @@
+1
```

Comment:
Ok, on the dev branch it's called `deployements/mainnet`, better to rename to `ethereum` as it has been renamed into the hardhat.config. Make sure it's the correct deployment coming from the production branch, mainnet directory

Reply:
mainnet is actually polygon :/